lib/queuePopulator/BucketFileLogReader.js

```diff
@@ -45,6 +46,7 @@ class BucketFileLogReader extends LogReader {
 
     getMetricLabels() {
         return {
+            origin: this._extensionNames,
```

Comment:
should we not "join" the strings in the array? (here or maybe better in the constructor)

```this._extensionNames.join(',')```

Reply:
`_extensionNames ` is a string containing that. The `join` is done when setting up the log source in the queue populator